content/release-notes/1.23.0.md

```diff
@@ -0,0 +1,28 @@
+---
+date: 2020-11-13T13:30:14-08:00
+linktitle: "1.23.0"
+title: "1.23.0"
+kubernetes: "1.17, 1.18, and 1.19"
+---
+
+{{<features>}}
+* Added new template function [KotsVersion](https://kots.io/reference/template-functions/static-context/#kotsversion)
+* Added [collectors](https://troubleshoot.sh/docs/collect/ceph/) and [analyzers](https://troubleshoot.sh/docs/analyze/ceph-status/) for Ceph.
+* Added a [collector](https://troubleshoot.sh/docs/collect/collectd/) for Collectd RRD files.
+{{</features>}}
+
+{{<changes>}}
+* Added a button for redeploying failed deployments.
+* User will be warned if an incompatible version of [Velero](https://kots.io/vendor/snapshots/overview/#velero-version-compatibility) is installed.
+* Improved description of what instance and application snapshots do.
+* Better error handling when [app configuration](https://kots.io/vendor/config/config-screen/) is being modified.
```

Comment:
Can you elaborate here?

Reply:
[It's just additional logging.](https://github.com/replicatedhq/kots/commit/1f91a86ba520226e4d78151beba1523d55f077ab)